src/app/components/Item/ItemHoldings.jsx

```diff
@@ -127,7 +127,7 @@ class ItemHoldings extends React.Component {
         />
       );
 
-      items = this.state.chunkedItems[this.state.page + 1];
+      items = this.state.chunkedItems[this.state.page - 1];
```

Comment:
items will be `undefined` if `this.state.chunkedItems` is an empty array. I don't think it will break anything as we have a fallback in <ItemTable>. But we might want to pass something rather then `undefined`?